### PR TITLE
feat: [SRM-13598]: Added slo identification stepper component

### PR DESCRIPTION
**Pull request by karansaraswat19**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by karansaraswat19** [comment]
retrigger jest

File: src/modules/85-cv/pages/slos/components/CVCreateDowntime/CVCreateDowntime.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    ...pathParams,
+    identifier,
+    lazy: true
+  })
+
+  useEffect(() => {
+    if (identifier) {
+      refetch()
+    }
+  }, [identifier, refetch])
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/slos/components/CVCreateDowntime/CVCreateDowntime.tsx:44]: NIT: we can avoid refetch

File: src/modules/85-cv/pages/slos/components/CVCreateDowntime/CVCreateDowntime.utils.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ */
+
+import * as Yup from 'yup'
+import type { Item } from '@harness/uicore/dist/components/ThumbnailSelect/ThumbnailSelect'
+import type { UseStringsReturn } from 'framework/strings'
+import type { DowntimeDTO } from 'services/cv'
+import { DowntimeForm, DowntimeFormFields } from './CVCreateDowntime.types'
+
+export const getDowntimeInitialFormData = (sloDowntime?: DowntimeDTO): DowntimeForm => {
+  return {
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/slos/components/CVCreateDowntime/CVCreateDowntime.utils.ts:15]: no need for extra braces , we can return directly

**Review by deepesh-ui — APPROVED** [review]

**Comment by praneshg239** [comment]
@karansaraswat19 please add sreenshot in the summary

**Review by harshilbits — APPROVED** [review]

**Pull request merged**